cmd/lukso/validator.go

```diff
@@ -59,6 +63,40 @@ func startValidator(ctx *cli.Context) (err error) {
 		return cli.Exit(fmt.Sprintf("❌  There was an error while starting validator: %v", err), 1)
 	}
 
+	passwordPipe.Close()
+
+	log.Info("⚙️  Please wait a few seconds while we validate your password...")
+	time.Sleep(time.Second * 10) // should be enough
+
+	logFile, err := getLastFile(ctx.String(logFolderFlag), validatorDependencyName)
+	if err != nil {
+		return cli.Exit(fmt.Sprintf("❌  There was an error while getting latest log file: %v", err), 1)
+	}
+
+	logs, err := os.ReadFile(ctx.String(logFolderFlag) + "/" + logFile)
+	if err != nil {
+		return cli.Exit(fmt.Sprintf("❌  There was an error while reading log file: %v", err), 1)
+	}
+
+	if strings.Contains(string(logs), wrongPassword) {
+		err = cfg.Read()
+		if err != nil {
+			return cli.Exit(fmt.Sprintf("❌  There was an error while reading config: %v", err), 1)
+		}
+
+		err = clientDependencies[cfg.Consensus()].Stop()
+		if err != nil {
+			return cli.Exit(fmt.Sprintf("❌  There was an error while stopping consensus: %v", err), 1)
+		}
+
+		err = clientDependencies[cfg.Execution()].Stop()
+		if err != nil {
+			return cli.Exit(fmt.Sprintf("❌  There was an error while stopping execution: %v", err), 1)
+		}
+
+		return cli.Exit("❌  Password incorrect, please restart and try again", 1)
```

Comment:
```suggestion
		return cli.Exit("❌  Incorrect password, please restart and try again", 1)
```

<img width="653" alt="image" src="https://github.com/lukso-network/tools-lukso-cli/assets/477945/d4785a03-6508-4674-8c47-8d1998317ce3">

There are 2 emojis, i think it is too much. But lets keep it like this.

Reply:
Done